tests/rom.rs

```diff
@@ -6,6 +6,18 @@ fn test_atari7800_rom() {
     assert_eq!(fmt, FileFormat::Atari7800Rom);
 }
 
+#[test]
+fn test_commmodore64_cartridge() {
+    let fmt = FileFormat::from_file("fixtures/rom/sample.crt").unwrap();
+    assert_eq!(fmt, FileFormat::Commodore64Cartridge);
+}
+
+#[test]
+fn test_commodore_program() {
+    let fmt = FileFormat::from_file("fixtures/rom/sample.prg").unwrap();
```

Comment:
Fixture needs to be moved in fixtures/executable

Reply:
fixed